client/reader/stream/index.jsx

```diff
@@ -588,6 +597,7 @@ export default connect(
 		const stream = getStream( state, streamKey );
 		const selectedPost = getPostByKey( state, stream.selected );
 		const streamKeySuffix = streamKey?.substring( streamKey?.indexOf( ':' ) + 1 );
+		const recommendedSites = getReaderRecommendedSites( state, 'seed?' ) || [];
```

Comment:
Probably good to change `seed?` to `discover-recommendations` or something? (and in the recieveRecommendedSites)